packages/e2e-tests/specs/editor/various/writing-flow.test.js

```diff
@@ -678,27 +678,11 @@ describe( 'Writing Flow', () => {
 		await page.keyboard.press( 'Tab' );
 		// Create the table.
 		await page.keyboard.press( 'Space' );
-		// Return focus after focus loss. This should be fixed.
-		await page.keyboard.press( 'Tab' );
 		// Navigate to the second cell.
 		await page.keyboard.press( 'ArrowRight' );
 		await page.keyboard.type( '2' );
 		// Confirm correct setup.
 		expect( await getEditedPostContent() ).toMatchSnapshot();
-		// The content should only have one tab stop.
```

Comment:
This is not the case. Each Tab key press will land in a cell so removed this. Let me know if I should do anything differently.